test/Heritable.test.js

```diff
@@ -63,6 +67,19 @@ contract('Heritable', function (accounts) {
     assert.isTrue(await heritable.heir() === heir);
   });
 
+  it('only heir can proclaim death', async function () {
+    const someRandomAddress = accounts[2];
+    await expectThrow(heritable.proclaimDeath({ from: owner }));
+    await expectThrow(heritable.proclaimDeath({ from: someRandomAddress }));
+  });
+
+  it('heir can\'t proclaim death if owner is death', async function () {
+    const heir = accounts[1];
+    await heritable.setHeir(heir, { from: owner });
+    heritable.proclaimDeath({ from: heir });
+    await expectThrow(heritable.proclaimDeath({ from: heir }));
```

Comment:
can we use `assertRevert` here?

Reply:
Sure, I've taken the opportunity to use it in all the PR changes.